subprojects/launcher/src/main/java/org/gradle/launcher/daemon/server/exec/ExecuteBuild.java

```diff
@@ -56,14 +56,12 @@ protected void doBuild(final DaemonCommandExecution execution, Build build) {
         try {
             BuildCancellationToken cancellationToken = execution.getDaemonStateControl().getCancellationToken();
             BuildRequestContext buildRequestContext = new DefaultBuildRequestContext(build.getBuildRequestMetaData(), cancellationToken, new DaemonConnectionBackedEventConsumer(execution));
-            if (!build.getParameters().isContinuous()) {
-                buildRequestContext.getCancellationToken().addCallback(new Runnable() {
-                    @Override
-                    public void run() {
-                        LOGGER.info(DaemonMessages.CANCELED_BUILD);
-                    }
-                });
-            }
+            buildRequestContext.getCancellationToken().addCallback(new Runnable() {
+                @Override
+                public void run() {
+                    LOGGER.info(DaemonMessages.CANCELED_BUILD);
```

Comment:
Why this change?

Reply:
Previously, only in continuous mode this message is printed because only continuous build can respond to cancellation. Now all builds can respond to cancellation so I think it's necessary to remove this limitation.